wire-benchmarks/build.gradle.kts

```diff
@@ -25,9 +25,9 @@ wire {
 }
 
 jmh {
-  jvmArgs = listOf("-Djmh.separateClasspathJAR=true")
-  include = listOf("""com\.squareup\.wire\.benchmarks\.EncodeBenchmark.*""")
-  duplicateClassesStrategy = DuplicatesStrategy.WARN
+  jvmArgs.addAll(listOf("-Djmh.separateClasspathJAR=true"))
+  includes.addAll(listOf("""com\.squareup\.wire\.benchmarks\.EncodeBenchmark.*"""))
+  duplicateClassesStrategy.set(DuplicatesStrategy.WARN)
```

Comment:
I had a lot of errors like "val cannot be set more than once" and "Received a String but expected a Propery<String>", etc...